app/views/admin/creator_settings/new.html.erb

```diff
@@ -41,6 +41,11 @@
   <%= render "form" %>
   <div class="crayons-field mt-6 align-left">
     <%= submit_tag "Finish", class: "crayons-btn btn--primary" %>
+    <% if @help_url %>
+      <a class="admin-help-button align-right" href="<%= @help_url %>" target="_blank" rel="noopener noreferer">
+        <%= inline_svg_tag("circle-question.svg", aria: true, title: "Forem Admin Guide", class: "crayons-btn crayons-btn--icon-rounded crayons-btn--s") %>
+      </a>
+    <% end %>
```

Comment:
I'm having a difficult time getting the "help" icon up to align with the finish button as shown in this design mockup, though I'm still working on it. If anyone has any suggestions on how to accomplish this, I would love to hear them!
![](https://user-images.githubusercontent.com/108287/116532347-6d033700-a8e0-11eb-8dae-88bf6164f682.png)

Reply:
I have a couple of thoughts on this: 

1. I know that the design looks like it aligns the help icon with the Finish button, but on a bigger screen with a centered form, that may look odd. My recommendation is to use the same placement as we currently have on admin (see screenshot below) so that it's consistent and users always click in the same place to find the icon. We can confirm this with @ludwiczakpawel who did the initial design. 

<img width="1528" alt="Screenshot 2021-11-09 at 13 59 46" src="https://user-images.githubusercontent.com/2786819/140920473-cfa0efaf-f471-4cd2-996e-5ac915ef27ef.png">

2. Based on the assumption that (1) is okay. My recommendation would be to not duplicate this code in both this file and the `admin.html.erb` but to rather use a partial and render the help icon through a partial. That way if we ever update the icon etc. we can do it one place.
```
    <% if @help_url %>
      <a class="admin-help-button align-right" href="<%= @help_url %>" target="_blank" rel="noopener noreferer">
        <%= inline_svg_tag("circle-question.svg", aria: true, title: "Forem Admin Guide", class: "crayons-btn crayons-btn--icon-rounded crayons-btn--s") %>
      </a>
    <% end %>
``` 

3. You can move the code above (the partial if you create one) into the `</main>` element next to the `forem-background`

4. Now, you may have noticed that [the class `admin-help-button`](https://github.com/forem/forem/blob/juliannatetreault/provide-help-on-creator-onboarding-page-14650/app/assets/stylesheets/admin.scss#L172-L187) is responsible for the placement and styling of the button. In your `creator_settings/new.html.erb` file the css for this class is not available. The reason being is that the css for this class is part of `admin.scss` and is only referenced in the [admin.html.erb](https://github.com/forem/forem/blob/juliannatetreault/provide-help-on-creator-onboarding-page-14650/app/views/layouts/admin.html.erb#L24). Since we're not using the "admin" layout, this class will not be available. 

Hence, my recommendation for (4) would be to:
a. Firstly, move some of the properties in this class name `admin-help-button` to use utility classes that we can place directly in the HTML. This seems to be more standard recently. 
b. Some of these classes will not be able to be moved to a utility class, for example, we cannot do calculations like these `calc(var(--header-height) + var(--su-3));` in a utility class, hence I would move this class to a stylesheet that can be referenced in both the admin and application layout. You could perhaps add it to `scaffolds.scss` which seems to be shared by both the admin and the application file. Maybe pop a comment to state why you've added it to the file too. 

My final recommendation is that you pull in `main` before continuing any further work as I _suspect_ there may be merge conflicts since I corrected a closing div in the file and moved some things around. 

If this feels confusing in any way, please feel free to ping me :)

Reply:
A partial definitely makes sense here as does the styling @Ridhwana proposes.

Reply:
@Ridhwana is correct - this help CTA does not have to be aligned with "Finish" button... It's just a coincidence on mockup :)

Reply:
Thanks so much for the input, @Ridhwana, @nickytonline, and @ludwiczakpawel -- I really appreciate it!

@Ridhwana I've implemented all of this feedback, but the utility classes + newly moved `.admin-help-button` class don't seem to work as intended--am I missing something here? I've mapped all of the classes that I could to utility classes and added them inline and then left the calculations untouched in `scaffolds.scss`, but the "help" button on both the Creator Onboarding settings page and throughout the Admin portal don't seem to have the styles applied as intended.